avocado/plugins/yaml_to_mux.py

```diff
@@ -293,10 +350,28 @@ def configure(self, parser):
             mux.add_argument("-m", "--mux-yaml", nargs='*', metavar="FILE",
                              help="Location of one or more Avocado"
                              " multiplex (.yaml) FILE(s) (order dependent)")
+            mux.add_argument('--mux-filter-only', nargs='*', default=[],
```

Comment:
PS: If you agree with this approach, I can create cards for this, the style fixes and some other non-critical improvements.